internal/exec/exec_test.go

```diff
@@ -0,0 +1,64 @@
+package exec
+
+import (
+	"os"
+	"testing"
+	"unsafe"
+
+	"github.com/Microsoft/hcsshim/internal/winapi"
+	"golang.org/x/sys/windows"
+)
+
+func TestExecProcess(t *testing.T) {
+	var procThreadSize uintptr
+	if err := winapi.InitializeProcThreadAttributeList(nil, 1, 0, &procThreadSize); err != nil && err != windows.E_NOT_SUFFICIENT_BUFFER {
```

Comment:
why do we not fail when the error is E_NOT_SUFFICIENT_BUFFER?

Reply:
First call always returns this iirc. https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-initializeprocthreadattributelist#remarks

Reply:
ah yes, okay